CHANGELOG.md

```diff
@@ -6,12 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 
+### Fixed
+
+- Fixes vscode-test running Thanks to [PR #3391](https://github.com/gitkraken/vscode-gitlens/pull/3391) by Nikolay ([@nzaytsev](https://github.com/nzaytsev))
```

Comment:
We use this format for outside contributors. GL team members can just simplify that.

```suggestion
- Fixes running unit tests and improves eslint configuration
```